WhenPushComesToShove/Assets/3-Scripts/Player Scripts/PlayerHitstun.cs

```diff
@@ -7,18 +7,24 @@ public class PlayerHitstun : Hitstun
 {   
     [SerializeField]
     private PlayerInputHandler inputHandler;
+    private bool stunned = false;
```

Comment:
@brschlegel Needed this because Unstun was being called every frame and preventing me from locking movement. If you prefer a different way of handling this, let me know and I'll change it.

Reply:
Only thing I would say is change Stun so it follows the same arrangement:
if(!stunned)
{
do the shit
}